tests/tracer/telemetry/test_writer.py

```diff
@@ -120,7 +122,20 @@ def test_add_app_started_event(mock_time, mock_send_request, telemetry_writer):
     assert httpretty.last_request().parsed_body == _get_request_body(payload, "app-started")
 
 
-def test_add_app_closing_event(mock_time, mock_send_request, telemetry_writer):
+def test_app_started_event_on_fork(telemetry_writer):
+    """asserts that app_started_event() is not sent after a fork"""
+    if os.fork() == 0:
```

Comment:
We don't check the child process?

Reply:
This test should test a child process only. I think os.fork returns a pid of zero for a child process. The test above tests what would occur in a parent/non-forked process